### DISCUSSION TITLE
Are there any other Map Styles (to download)? Modify colors possible?

### DISCUSSION THREAD

**Discussion by pstein** [osmandapp/OsmAnd#13936] [Q&A]
I don't like all the available Map Styles.

Sometimes  e.g. highways are colored in purple sometimes in pink.

Big streets are sometimes colored in yellow sometimes in orange

I want to hide county border dashed lines

Are there any additional Map Styles for download?

How can I change the assigned colors for only some of the map components?

**Comment by StarlightRunner** [comment]
it's entirely possible to create your own map style [[For example, here's mine.](https://litter.catbox.moe/mlqlih.png)], but it takes manual editing of the render.xml files generated for each map style, as well as an understanding of OSM in general [what highways are, especially].

One helpful resource is this [Pokemon GO nest-finder](
https://gist.github.com/Rofang/4718ddba755246d94fedd052f748acbc) (PoGO defining "biomes" via actual landmarks). there was another, slightly newer one as well, which I'll link once I find it.
